### DISCUSSION TITLE
Templ as a general text template system?

### DISCUSSION THREAD

**Discussion by ltdangle** [a-h/templ#915] [General]
Hi all,
Apologies if has been asked before, but has there been any discussion on using Templ as regular text-based template system? I wanted to use Templ for emails, for example, but since it is html only, the text would come out without line breaks, etc. 

I Like Templ, would like to use it  for text mangling besides html...

**Comment by joerdav** [comment]
Hey, this has been discussed before yes, the following comment explains the current stance towards that:

https://github.com/a-h/templ/issues/385#issuecomment-1874603038

In short, it's currently not within scope for templ to write plain text or other formats than HTML. That being said, you could write your emails as HTML using templ, and use either a `<br />` or `<p>` tags.

I hope this helps!

**Reply by ltdangle** [reply]
Appreciate your reply (I see the light)!
